create_pdfs.py

```diff
@@ -37,8 +37,9 @@ def parse_args(args=sys.argv[1:]):
     parser.add_argument('-i', '--issued-date', help='optional issue date')
     parser.add_argument(
         '-T',
```

Comment:
I'd say it's a pretty relevant change. The whole point of the way we're writing this feature is to allow for more general "designations" to be passed.